streams/src/main/java/org/apache/kafka/streams/errors/TaskMigratedException.java

```diff
@@ -28,43 +28,40 @@
 
     private final static long serialVersionUID = 1L;
 
-    private final Task task;
+    private final TaskId taskId;
```

Comment:
@abbccdda My current proposal is to move all the PRs into a separate `K9113-base` branch which is periodically rebased on trunk, and have consecutive PRs against that `base` branch, so that we do not have to make sure each PR leaves the branch in a perfect state (system tests, unit tests, etc). So I think in this case this PR's size is okay -- as I said although it is big in LOC the non-testing changes are not so big, and this change is actually needed since I refactored the place when record collector etc are constructed.